cc/README.md

```diff
@@ -69,7 +70,7 @@ To run Minigo with a 9x9 model:
 
 ```shell
 bazel build --define=board_size=9 -c opt cc:main
-bazel-bin/cc/main --model=$MODEL_PATH
+bazel-bin/cc/main --model=$MODEL_PATH --mode=selfplay
 ```
 
 To run Minigo with a 19x19 model:
```

Comment:
line three down from here should also get --mode=selfplay (or gtp)

Reply:
Thanks. Will fix.